apps/hexagon_api/CMakeLists.txt

```diff
@@ -88,3 +88,6 @@ ExternalProject_Add_Step(hexagon_tvm_runtime_rpc copy_binaries
   DEPENDEES install
 )
 
+# Copy android_bash template file
```

Comment:
Just a style suggestion (feel free to ignore this): The comment line seems unnecessary, because the following cmake lines are really easy to grok.

Reply:
removed it.